models/contracts/optimism/contracts_optimism_contract_overrides.sql

```diff
@@ -0,0 +1,21 @@
+{{ 
+  config(
+    schema = 'contracts_optimism', 
+    alias='contract_overrides',
+    post_hook='{{ expose_spells(\'["optimism"]\',
```

Comment:
maybe add a uniqueness check on "contract_address." We should never have duplicates.